src/components/Dashboard/FilingHistoryList/PendingFiling.vue

```diff
@@ -2,8 +2,8 @@
   <div v-if="filing" class="pending-filing-details body-2">
     <h4>Filing Pending</h4>
 
-    <p>This {{title}} is paid, but the filing has not been completed by the BC
-      Registry yet. Some filings may take longer than expected.</p>
+    <p>This {{title}} is paid, but the filing has not been completed by BC Registries yet.
```

Comment:
The Business Registry?

Reply:
Done.

Reply:
Ok to proceed with "by the Business Registry"

Reply:
Done.